Makefile

```diff
@@ -6,14 +6,14 @@ MANPAGE = $(ASSETS_DIR)/alacritty.man
 TERMINFO = $(ASSETS_DIR)/alacritty.info
 COMPLETIONS_DIR = $(ASSETS_DIR)/completions
 COMPLETIONS = $(COMPLETIONS_DIR)/_alacritty \
-			  $(COMPLETIONS_DIR)/alacritty.bash \
-			  $(COMPLETIONS_DIR)/alacritty.fish
+			$(COMPLETIONS_DIR)/alacritty.bash \
+			$(COMPLETIONS_DIR)/alacritty.fish
```

Comment:
Tabs always make this unnecessarily difficult to see, but are those three tabs of indentation? Why?

You'd think there should be one tab of indentation at most beyond the previous element, past that we're talking about alignment which should be done with spaces?